doc/sphinx/source/code.rst

```diff
@@ -0,0 +1,111 @@
+.. _Code:
```

Comment:
Names are hard. Calling this section 'Code' feels off, but I'm not sure of a better name. 'libCEED API' perhaps?

Reply:
"API Documentation" or "API Reference" is a common term for that. It should perhaps be split into sub-pages (quite large now).

Reply:
Done